README.adoc

```diff
@@ -82,15 +82,21 @@ DHCP network is added by default and a warning is displayed.
 1.  Run the plugin to get environment variables and certificates:
 +
 ----------------------------------------------------------------------------
-$ vagrant service-manager env docker
-# Set the following environment variables to enable access to the
-# docker daemon running inside of the vagrant virtual machine:
-export DOCKER_HOST=tcp://172.28.128.4:2376
-export DOCKER_CERT_PATH=/foo/bar/.vagrant/machines/default/virtualbox/docker
-export DOCKER_TLS_VERIFY=1
-export DOCKER_API_VERSION=1.21
-# run following command to configure your shell:
-# eval "$(vagrant service-manager env docker)"
+ # docker env:
```

Comment:
Ack! this change was made as per input given to me earlier. My bad! I should have tested before making the change. Will make changes as mentioned.